Sources/Rendering/OpenGL/VolumeMapper/index.js

```diff
@@ -568,6 +580,30 @@ function vtkOpenGLVolumeMapper(publicAPI, model) {
       // specify the planes in view coordinates
       program.setUniform3f(`vPlaneNormal${i}`, normal[0], normal[1], normal[2]);
       program.setUniformf(`vPlaneDistance${i}`, dist);
+
+      if (actor.getProperty().getUseLabelOutline()) {
+        const image = model.currentInput;
+        const worldToIndex = image.getWorldToIndex();
+
+        program.setUniformMatrix('vWCtoIDX', worldToIndex);
+
+        // Compute the view coordinate to world coordinate transformation matrix.
+        model.viewToWorld = mat4.create();
+        mat4.invert(model.viewToWorld, keyMats.wcvc);
+
+        const size = publicAPI.getRenderTargetSize();
+        const aspect = size[0] / size[1];
+        const viewToWorld = ren
+          .getActiveCamera()
+          .getCompositeProjectionMatrix(aspect, -1.0, 1.0);
+
+        mat4.invert(viewToWorld, viewToWorld);
+        mat4.transpose(viewToWorld, viewToWorld);
+
+        program.setUniformMatrix('VCWCMatrix', viewToWorld);
```

Comment:
Should be named DCWCMatrix I think as it is from device to world unless I am missing something